runtime/doc/job_control.txt

```diff
@@ -102,23 +102,23 @@ function. Here's a more object-oriented version of the above:
 >
     let Shell = {}
 
-    function Shell.on_stdout(job_id, data)
+    function Shell.on_stdout(job_id, data) dict
```

Comment:
In my plugins I am trying to follow the following convention: `function dict.func(…)` means that function will not be using `self` dictionary, `dict` in signature means that `self` will be used. This is because my plugins use dictionary functions like `s:F.foo` everywhere, except for some user (not developer!) API or for technical reasons (in cases when anonymous functions are way too tricky to use).